content/en/docs/ops/app-health-check/index.md

```diff
@@ -121,7 +121,7 @@ The configuration changes above (by Helm or by the configuration map) effect all
 
 <!-- Add samples YAML or kubectl patch? -->
 
-Rather than install Istio with different Helm option, you can annotate Pod with `sidecar.istio.io/rewriteAppHTTPProbers: "true"`.
+Rather than install Istio with different Helm option, you can annotate Pod with `sidecar.istio.io/rewriteAppHTTPProbers: "true"` under `spec.template.metadata.annotations`.  If you annotate under `metadata.annotations`, the annotation is ignored.
```

Comment:
This wording assumes our customers use `Deployment`, which maybe 90% true for most cases. However, as I pointed out in the issue, it's possible other resources built on top of the Pod, like `ReplicaSet` etc. So it's not 100% accurate.

sidecar injector guru, @liamawhite  @ayj thoughts?

Reply:
Hmm, I think it'll be fine if you mention "For `Deployment`, annotations should be in ....`.